tests/foreman/api/test_docker.py

```diff
@@ -135,6 +136,20 @@ def _create_repository(product, name=None, upstream_name=None):
     ).create()
 
 
+class RplevkaTestCase(APITestCase):
+    """test class
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(RplevkaTestCase, cls).setUpClass()
+
+    @tier1
+    @skip_if(1 == 1, 'skipping due one is one')
+    def test_rplevka(self):
+        pass
+
+
 class DockerRepositoryTestCase(APITestCase):
```

Comment:
should go to robottelo/tests
- one case for skipping
- one case for not skipping